registry/finders.go

```diff
@@ -217,6 +217,26 @@ type AppsListOptions struct {
 	Filters map[string]string
 }
 
+func GetPendingVersions(c *Space) ([]*Version, error) {
+	db := c.dbPendingVers
+	rows, err := db.AllDocs(ctx)
+	if err != nil {
+		return nil, err
+	}
+	defer rows.Close()
+
+	versions := make([]*Version, 0)
```

Comment:
nit: zero initialization is the "common" way to do this `var versions []*Version`